plugins/generator-1.20.4/neoforge-1.20.4/utils/triggers.java.ftl

```diff
@@ -12,7 +12,8 @@
 				"y": "entity != null ? entity.getY() : 0.0",
 				"z": "entity != null ? entity.getZ() : 0.0",
 				"entity": "entity",
-				"world": "level instanceof Level ? (LevelAccessor) level : null"
+				"world": "level instanceof Level ? (LevelAccessor) level : null",
+				"itemstack": "itemstack"

```

Comment:
How is this related to this PR?

Is this some bug you have discovered? 

In this case, this would deserve own PR and backport to other generators